app/[locale]/roadmap/_components/ReleaseCarousel.tsx

```diff
@@ -0,0 +1,272 @@
+            >
+              <CarouselContent>
+                {releasesData.map((release, index) => {
+                  const releaseDate = new Date(release.releaseDate)
+                  const nextRelease =
+                    releaseDate > todayDate && releaseDate <= twoMonthsFromNow
+                  const labelType =
+                    releaseDate < todayDate
+                      ? 1
+                      : releaseDate < twoMonthsFromNow
+                        ? 2
+                        : 3
+
+                  return (
+                    <CarouselItem
+                      key={release.releaseName}
+                      className="w-full md:basis-1/3"
+                    >
+                      <div className="flex w-full flex-col items-center justify-center gap-3">
+                        <div className="mb-3 !h-6">
+                          {labelType === 1 && (
+                            <div
+                              className={cn(
+                                "w-fit rounded-lg bg-primary-low-contrast px-2 py-1",
+                                currentIndex !== index && "hidden"
+                              )}
+                            >
+                              <p className="text-sm font-bold">In production</p>
+                            </div>
+                          )}
+                          {labelType === 2 && (
+                            <div
+                              className={cn(
+                                "w-fit rounded-lg bg-warning-light px-2 py-1",
+                                currentIndex !== index && "hidden"
+                              )}
+                            >
+                              <p className="text-sm font-bold">Coming soon</p>
+                            </div>
+                          )}
+                          {labelType === 3 && (
+                            <div
+                              className={cn(
+                                "w-fit rounded-lg bg-roadmap-upgrade-card-gradient-hover px-2 py-1",
+                                currentIndex !== index && "hidden"
+                              )}
+                            >
+                              <p className="text-sm font-bold">
+                                In development
+                              </p>
+                            </div>
+                          )}
+                        </div>
+                        <div className="flex w-full items-center justify-center text-center">
+                          <div
+                            className={cn(
+                              "flex h-1 flex-1",
+                              index !== 0
+                                ? nextRelease
+                                  ? "bg-gradient-to-r from-primary to-primary-low-contrast"
+                                  : release.releaseDate <
+                                      new Date().toISOString().split("T")[0]
```

Comment:
we could reuse some of the vars already defined and have a safer comparison with numbers
```suggestion
                                  : releaseDate.getTime() < todayDate.getTime()
```